### DISCUSSION TITLE
Commercial Usage

### DISCUSSION THREAD

**Discussion by bulutenesemre** [meteostat/meteostat-python#48] [General]
Hey good devs,

I want to use meteostat-python in a product. I can see from MIT License there is no problem about commercial usage , on the other hand it might be problem because of data providers right? 

Can we make it clearify and are there anyone use it for commercial usage?

**Comment by clampr** [comment]
I have no problem with commercial use of Meteostat data. The only hiccup is WMO resolution 40 which doesn't allow data to be redistributed commercially "as is". However, I don't think anyone wants to take the data as it is published by NOAA and just put a price tag on it. Usually, companies will use the data internally or ship it as part of another product. Therefore, I don't see an issue with the resolution. But as I'm not a lawyer, I'm redistributing data under a NonCommercial CC. If anyone knows more about the legal background, I would love to change the license so it officially suppprts commercial use.
